package.json

```diff
@@ -54,16 +56,19 @@
     "@folio/eslint-config-stripes": "^3.2.0",
     "@folio/stripes-cli": "^1.5.0",
     "babel-eslint": "^9.0.0",
+    "babel-plugin-lodash": "^3.3.4",
     "babel-register": "^6.16.3",
     "chai": "^4.1.2",
     "eslint": "^5.5.0",
+    "lodash-webpack-plugin": "^0.11.5",
```

Comment:
The plugins will need to applied as dependencies, rather than devDependencies, otherwise they won't be installed when yarn installing a platform.

Reply:
Got it, moved!

Reply:
Wouldn't we normally want these as devDependencies since they aid in the Babel transpile process but aren't actually used in the bundle?

Reply:
In a typical app that manages its own build, yes. FOLIO, being distributed across multiple repos, has its build logic and dependencies consolidated in stripes-core.  However, devDependencies are only ever installed for the root package.json.  For FOLIO, the `platform-*` repos are the root of a build (and sometimes `ui-*` modules in a dev environment). Therefore in order to have Babel, Webpack, etc. centrally defined and installed, they must be declared as regular dependencies of stripes-core.

Reply:
Makes sense. Thanks for explaining!